src/getTime.js

```diff
@@ -18,7 +18,8 @@
  * @returns {string} - valid time
  */
 function getTime(str) {
-  // write code here
+  const regExp = /\b2[0-3]{1}:[0-5]{1}\d\b|\b[01]{1}\d:[0-5]{1}\d\b/;
+  return (regExp.exec(str) !== null) ? regExp.exec(str)[0] : '';
```

Comment:
`exec ` - may be a hard and long operation, better to write the result in variable